pkgs/applications/networking/cluster/atmos/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "atmos";
+  version = "1.4.28";
+
+  src = fetchFromGitHub {
+    owner = "cloudposse";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-AqW//+x1tKzm5dJirXCLFuDBTk3dHHw7c6vwsVsI37U";
+  };
+
+  vendorSha256 = "sha256-bepv8Ns8vcpLpmO7fsRgGijB9B6kQ5HhxUhIuEpGtOY=";
+
+  ldflags = [ "-X 'github.com/cloudposse/atmos/cmd.Version=${version}'" ];
```

```suggestion
  ldflags = [ "-X github.com/cloudposse/atmos/cmd.Version=v${version}" ];
```
needs a v prefix

```
/ # ./atmos_1.4.28_linux_amd64 version
v1.4.28
```

You can put single quotes around it but it's not strictly necessary unless there are spaces

---

Fixed!